steps/steps-chatwork-message/0.0.3/step.yml

```diff
@@ -0,0 +1,59 @@
+title: Send a Chatwork message
+summary: |
+  Send a Chatwork message
+description: |
+  Send a Chatwork message from your Bitrise build!
+website: https://github.com/monolithic-adam/bitrise-step-chatwork-message
+source_code_url: https://github.com/monolithic-adam/bitrise-step-chatwork-message
+support_url: https://github.com/monolithic-adam/bitrise-step-chatwork-message/issues
+published_at: 2019-10-10T17:24:35.832487+09:00
+source:
+  git: https://github.com/monolithic-adam/bitrise-step-chatwork-message.git
+  commit: f542cc170fd3f448138c0287fe2ef3e479beefc4
+type_tags:
+- notification
+toolkit:
+  go:
+    package_name: github.com/monolithic-adam/bitrise-step-chatwork-message
+is_requires_admin_user: false
+is_always_run: true
+is_skippable: true
+inputs:
+- api_token: null
+  opts:
+    description: |
+      **An api\_token input is required.**
+
+      To get your **API token** visit: https://www.chatwork.com/service/packages/chatwork/subpackages/api/token.php
+    is_required: true
+    is_sensitive: true
+    title: Chatwork API Token
+- opts:
+    description: |
+      **An room\_id input is required.**
+
+      To get your **Room ID** look at the #!rid in your URL
```

Comment:
Please add more details about this, make it more clear.

Reply:
added a bit more detail and a URL for help